src/components/RatingQuestion/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+const RatingQuestion = ({ initialRating, ratingCount, ratingEmoji }) => {
+  const [rating, setRating] = useState(initialRating);
+  const [emoji] = useState(initEmoji);
+  const [hover, setHover] = useState(null);
+  const emojiElement = [...Array(ratingCount || 5)];
+
+  function initEmoji(ratingEmoji) {
+    switch(ratingEmoji) {
+      case 'heart':
+        return '❤️';
+      case 'smiley':
+        return '🙂';
+      case 'star':
+        return '⭐';
+      default:
+        return '👍🏻';
+    }
+  }
+
+  return (
+    <div className="rating">
+      { emojiElement.map((thumb, index) => {
```

Comment:
@noor-ahmed As we aren't use this variable `thumb`, should we make it like this `_thumb` ?

Reply:
As there are more types now, should we think of a more generic name other than `thumb`?